docs/source/documentation/team/wow-2.html.md.erb

```diff
@@ -0,0 +1,222 @@
+
+## Who Are We?
+
+Find our team on [MoJ People Finder](https://peoplefinder.service.gov.uk/teams/data-platform-apps-and-tools)
+
+## What are our Core Values?
+
+- **Collaboration:** Foster a culture of open communication and collective problem-solving
+- **Quality:** Uphold high standards in our work, ensuring reliability and efficiency
+- **Innovation:** Embrace new ideas and technologies that propel us and our platforms forward
+- **Transparency:** Maintain clear visibility into our processes, decisions, and progress
+
+## Team Roles
+- **Team Members:** Comprising a mix of developers, data engineers, and platform engineers, responsible for the delivery of tasks and active participation in all Agile ceremonies
+- **Product Manager (PM):** Oversees the product backlog, ensuring it aligns with stakeholder needs and business objectives
+- **Delivery Manager (DM):** Facilitates the Agile process, removing blockers and ensuring timelines are met
+
+## Communication and Delivery
+- **Daily Stand-ups:** A quick sync to share updates, plans, and blockers
+- **Ceremonies:** How we make sure our team stays aligned as we work to deliver during Agile Sprints.
+
+## Ceremonies
+
+### Schedule
+
+This team uses **Bi-weekly Sprints:** This is our timeframe for planning, executing, and reviewing work.
+
+The team has the following ceremonies:
+
+| Ceremony                                      | Occurence | Day             | Time          |
+| --------------------------------------------- | --------- | --------------- | ------------- |
+| [Stand Up](#stand-up)                         | Daily     | Monday - Friday | 10:00 - 10:15 |
+| [Backlog Management](#backlog-management)     | Weekly    | Thursday        | 14:00 - 15:00 |
+| [Refinement](#refinement)                     | Weekly    | Tuesday         | 11:00 - 12:00 |
+| [Sprint Planning](#sprint-planning)           | Bi-Weekly | Thursday        | 10:00 - 11:00 |
+| [Sprint Review](#sprint-review)               | Bi-Weekly | Thursday        | 10:00 - 11:00 |
+| [Sprint Retrospective](#sprint-retrospective) | Bi-Weekly | Tuesday         | 14:00 - 15:00 |
+| [Show & Tell](#show-and-tell)                 | Bi-Weekly | Thursday        | 15:00 - 15:30 |
+
+### Stand Up
+
+Our stand-ups run daily at 10:00 and last for fifteen minutes. We use [Slack Huddles](https://app.slack.com/huddle/T02DYEB3A/C04M8224WCV) and follow one of two formats:
+
+- **Walk The Board**
+
+Starting with the “Done” column we discuss any active issues on our [GitHub Projects Board](https://github.com/orgs/ministryofjustice/projects/27/views/18).
+
+- **Yesterday, Today, Blocked**
+
+A round robin of each team member discussing their issues and any blockers.
+
+Standup each day will be run by the person on support that day (A.K.A. "Support Lead")
+
+### Backlog Management
+
+**Purpose:**
+Backlog Management is where we ensure our backlog is up to date and ready for refinement. It's where we:
```

Comment:
Is this just for the product/delivery managers, and maybe leads, rather than full team? Or perhaps we dont want to specify here?